### PR TITLE
i18n(es): Update `images.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `images.mdx` based on these 2 commits 1d8d51dca8ebd9df74ca627280aaba020d4474ce & a4b7585d27e75cd0e5e351e0fe34ab1dadf4c407

### CODE REVIEW

**Review by ElianCodes — COMMENTED** [review]
Just left one little suggestion, everything else looks good to me! Thanks @dreyfus92!

File: src/content/docs/es/guides/images.mdx [hunk 1]
```diff
@@ -31,7 +31,7 @@ Astro puede obtener tus datos de forma remota utilizando APIs o mostrar imágene
 
 En archivos `.astro`, las imágenes locales deben ser importadas al archivo para poder ser utilizadas. Las imágenes remotas y las imágenes en `public/` no requieren ser importadas.
 
-Importa y utiliza el componente incorporado `<Image />` de Astro para imágenes optimizadas usando `astro:assets`. Alternativamente, la sintaxis de Astro admite escribir directamente una etiqueta HTML `<img>`, lo que omite el procesamiento de imágenes.
+Importa y utiliza el componente incorporado `<Image />` de Astro para imágenes optimizadas usando `astro:assets`. Alternativamente, la sintaxis de Astro admite escribir directamente una etiqueta HTML `<img>`, lo que omite el procesamiento de imágenes
```
[the remark this answers is not in the document]
Comment (dreyfus92) [reply src/content/docs/es/guides/images.mdx:34]: Hey Elian, it was removed here https://github.com/withastro/docs/commit/a4b7585d27e75cd0e5e351e0fe34ab1dadf4c407#diff-9ea5d63bc29a5a013378b352f2b77fffd61f773692ff6c329a3cc9292552ac23R35

File: src/content/docs/es/guides/images.mdx [hunk 2]
```diff
@@ -696,4 +704,23 @@ Si estabas utilizando la integración de imágenes en Astro v2.x, completa los s
 
 Ahora puedes declarar una imagen asociada para una entrada de colecciones de contenido, como la imagen de portada de una entrada de blog, en tu metadatos usando su ruta relativa a la carpeta actual.
 
-El nuevo helper `image` para colecciones de contenido te permite validar los metadatos de la imagen utilizando Zod. Aprende más sobre [cómo usar imágenes en colecciones de contenido](/es/guides/images/#imágenes-en-colecciones-de-contenido)
\ No newline at end of file
+El nuevo helper `image` para colecciones de contenido te permite validar los metadatos de la imagen utilizando Zod. Aprende más sobre [cómo usar imágenes en colecciones de contenido](/es/guides/images/#imágenes-en-colecciones-de-contenido)
+
+### Navegando por las importaciones de imágenes en Astro v3.0
+
+En Astro v3.0, si tienes que conservar el antiguo comportamiento de importación para las imágenes y requieres una representación de cadena de la URL de la imagen, añade `?url` al final de la ruta de la imagen al importarla. Por ejemplo:
```
Comment (Waxer59) [inline src/content/docs/es/guides/images.mdx:711]:
```suggestion
En Astro v3.0, si tienes que conservar el antiguo comportamiento de importación para las imágenes y requieres una representación de tipo string de la URL de la imagen, añade `?url` al final de la ruta de la imagen al importarla. Por ejemplo:
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by ElianCodes — APPROVED** [review]
Good to go! Thanks @dreyfus92 and @Waxer59 for the review!

**Pull request merged**